internal/pkg/cli/cmd/dbt/cmd.go

```diff
@@ -1,22 +1,22 @@
-package dbt

```

Comment:
Fixnute CS v kode od Honzu (musime opravit linter v CI).

Reply:
Jo, VSCode mi vytvářel nové soubory s CRLF, už jsem si to přenastavil na LF